docs/vms/vms.md

```diff
@@ -122,6 +122,21 @@ Once done with Linux, shut down the VM and restore the parameter to its original
 $ xe vm-param-set uuid=VM-UUID platform:device_id=0002
 ```
 
+## 🐧Linux VMs
+### Old Linux kernels and XFS
+
+Performance issues have been observed after a live migration, under heavy disk activity, for VMs running kernels prior to v4.20 and using the XFS file system.
+
+XFS seems to have better performances with recent kernels but for older ones we recommend to use another journaled file system like EXT4.
+
+### Ubuntu, UEFI, and Xen Orchestra console
+
+This only concerns `UEFI` VMs running `Ubuntu` distribution. Also note that a default `Ubuntu` installation works just fine.
+
+The default Ubuntu installation includes a package name `linux-modules-extra` containing the only driver (`bochs`) suitable for the VGA cards emulated by Qemu for the Xen Orchestra console. Removing this package leads to a scrambled display in the console so please don't do so.
```

Comment:
```suggestion
The default Ubuntu installation includes a package named `linux-modules-extra` containing the only driver (`bochs`) suitable for the VGA cards emulated by Qemu for the Xen Orchestra console. Removing this package leads to a scrambled display in the console so please don't do so.
```

Reply:
Fixed, thanks.

Also changed the chapter title for this one.